charts/sourcegraph-migrator/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: Chart for running Sourcegraph migrator
 type: application
 
 # Chart version, separate from Sourcegraph
-version: 0.1.0
+version: "3.39.0"
```

Comment:
this looks dodge?

Reply:
cc @caugustus-sourcegraph

Reply:
Nope, this is ok! We're changing to a new versioning scheme starting with this release that syncs up with how our other deployments are versioned.